kubectl-cloudflow/verify/blueprint_spec_test.go

```diff
@@ -0,0 +1,625 @@
+				inletSchema0: FOO,
+				inletSchema1: BAR,
+				outletSchema: FOO,
+
+			})
+			var ingressRef = ingress.ref("foo", nil)
+			var mergeRef   = merge.ref("bar", nil)
+			var blueprint = Blueprint{}
+			blueprint = blueprint.define([]StreamletDescriptor{ingress, merge}).use(ingressRef).use(mergeRef)
+			var connected = blueprint.connect(ingressRef.name, mergeRef.in0())
+
+			Expect(len(connected.UpdateGlobalProblems())).Should(Equal(1))
+			Expect(connected.UpdateGlobalProblems()).Should(ConsistOf(
+				UnconnectedInlets{
+					unconnectedInlets: []UnconnectedInlet{{streamletRef: "bar", inlet: merge.Inlets[1]}},
+				},
+			))
+		})
+	})
+
+	Context("by default", func() {
+		It("not fail verification with UnconnectedInlets for already reported IllegalConnection and IncompatibleSchema problems", func() {
+			var ingress = randomStreamlet()
+			var processor = randomStreamlet()
+			var egress = randomStreamlet()
+			ingress = ingress.asIngress("out", "foo", FOO)
+			processor = processor.asProcessor("out", "bar", "in", "foo", FOO, BAR)
+			pattern := "^1,65535$"
+			egress = egress.asEgress("in", "bar", BAR).withConfigParameters(
+				[]domain.ConfigParameterDescriptor{{
+					Key: "target-uri",
+					Description: "",
+					Type: "string",
+					Pattern: &pattern,
+					DefaultValue: nil,
+				}})
+			var blueprint = Blueprint{}
+			var ingressRef    = ingress.randomRef()
+			var processor1Ref = processor.randomRef()
+			var processor2Ref = processor.randomRef()
+			var egress1Ref    = egress.randomRef()
+			var egress2Ref    = egress.randomRef()
+			blueprint = blueprint.define([]StreamletDescriptor{ingress, processor, egress}).use(ingressRef).use(processor1Ref).use(processor2Ref).use(egress1Ref).use(egress2Ref)
+			blueprint = blueprint.connect(ingressRef.out(), processor1Ref.in()).connect(ingressRef.out(), processor2Ref.in()).connect(processor1Ref.out(), egress1Ref.in())
+			blueprint = blueprint.connect(processor2Ref.out(), egress1Ref.in()).connect(ingressRef.out(), egress2Ref.in())
+			blueprint = blueprint.upsertStreamletRef(egress1Ref.name, nil, nil).upsertStreamletRef(egress2Ref.name, nil, nil)
+			problems := blueprint.UpdateGlobalProblems()
+			outPortPath := "out"
+			inPortPath := "in"
+			Expect(problems).Should(ConsistOf(
+				IllegalConnection{
+					outletPaths: []VerifiedPortPath{
+						{streamletRef:processor1Ref.name,  portName: &outPortPath},
+						{streamletRef:processor2Ref.name, portName: &outPortPath},
+				},
+				inletPath: VerifiedPortPath{streamletRef:egress1Ref.name, portName: &inPortPath},
+				},
+				IncompatibleSchema{
+					outletPortPath: VerifiedPortPath{streamletRef:ingressRef.name,  portName: &outPortPath},
+					inletPath: VerifiedPortPath{streamletRef:egress2Ref.name,  portName: &inPortPath},
+				}))
+
+		})
+	})
+
+	Context("by default", func() {
+		It("should fail verification for configuration parameters with invalid validation patterns", func() {
+			pattern := `^.{1,65535\K$`
```

Comment:
We have an issue here since regex are language specific. In the current implementation java regex are used for both verification and use. If we go polyglot this does not make much sense and also does not make much sense if the streamlet is written in java/Scala but developer needs to express his configuration regex in Go.